reports/macbox7.md

```diff
@@ -0,0 +1,489 @@
+---
+layout: report
+permalink: /GSoC/2018/StatusReport/macbox7
```

Comment:
remove this and layout:

add date, categories, etc, as shown at https://github.com/coala/projects/pull/633/files#diff-f1164fdbd5814b6d270a59c0f9e3939e